android-sdk/android/src/main/kotlin/io/logto/sdk/android/LogtoClient.kt

```diff
@@ -41,6 +44,8 @@ open class LogtoClient(
 
     protected var oidcConfig: OidcConfigResponse? = null
 
+    protected var jwks: JsonWebKeySet? = null
```

Comment:
Do we have to update our SDK convention page, adding this property to LogtoClient? As of right now, other platform SDKs don't seem to have this.

Reply:
I think we don't. because this is for caching purposes, and maybe other platform SDK will do caching in other layers. e.g. like in the network session.

Reply:
OK, so it's a Kotlin specific thing? I'm good with leaving it as-is for now.